### PR TITLE
clean: rename wrapper to eventer & wrap to event

### PR DESCRIPTION
**Pull request by i-am-yuvi**
## Describe your changes
The `wrapper` field is renamed to `eventer` in the same way `wrap` is changed to `event`.

## Issue ticket number and link
#18 

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have compiled and built the program.

### Compiling Program
Compiling Code

### Running extractor
running extractor

### Running Logger
running logger

### CODE REVIEW

File: protobuf/proto-types/eventer.proto [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+package eventer;
 
 import "net_msg.proto";
 import "net_conn.proto";
 import "addrman.proto";
 import "mempool.proto";
 import "validation.proto";
 
-message Wrapper {
+message Eventer {
```
Comment (0xB10C) [inline protobuf/proto-types/eventer.proto:11]:
```suggestion
message EventMsg {
```

I don't think [Eventer](https://dictionary.cambridge.org/dictionary/english/eventer) is a proper English word describing this message. I would just use `EventMsg` here.

**Review by 0xB10C — CHANGES_REQUESTED** [review]

**Comment by 0xB10C** [comment]
Also make sure to run `cargo fmt` before commiting. Otherwise the CI will complain :)

https://github.com/0xB10C/peer-observer/actions/runs/9961953175/job/27547091674?pr=24

**Comment by 0xB10C** [comment]
> ### Compiling Program
> Compiling Code

No need to include such screenshots. It's the task of a reviewer (with the help of CI) to verify the changes.

**Comment by 0xB10C** [comment]
Oh and we should run https://github.com/0xB10C/peer-observer/blob/master/protobuf/gen-python.sh to regenerate the Python protobuf definitions. You can add the changes in a separate commit.

File: protobuf/proto-types/eventer.proto (hunk 1 above)
Comment (i-am-yuvi) [reply protobuf/proto-types/eventer.proto:11]: Makes sense!!👍

**Comment by i-am-yuvi** [comment]
Okay I shall make the changes!!

**Comment by 0xB10C** [comment]
Looks good! Can you change this up into two commits similar to this:

- clean: rename wrapper to event_msg & wrap to event
- update: protobuf python-types

Having the second commit split off allows me to run `gen-python.sh` myself and see if we get any differences.

Once that's done, I think this is ready for merge.

**Comment by i-am-yuvi** [comment]
Surely!!!

**Pull request closed without merging**